playbooks/setup_jenkins_slave.yml

```diff
@@ -26,11 +37,17 @@
         ClientAliveInterval: 15
         Compression: "yes"
   tasks:
+    - meta: flush_handlers
```

Comment:
Flushing the handlers reloads sshd, right? If so, do we need to add a wait here to verify that sshd is ready to receive connections again? With Ansible 2.3 I'm thinking something like https://github.com/openstack/openstack-ansible/blob/77ac88197c223968e845e57706b81af24d6aa443/playbooks/lxc-containers-create.yml#L28-L33

Reply:
The reason for adding this, to capture, is because at present the restart happens after the slave is connected, which we think is contributing to the slave disconnection that we're facing.

Reply:
done.